rs/bitcoin/ckbtc/minter/src/state.rs

```diff
@@ -395,6 +404,20 @@ pub struct CkBtcMinterState {
     /// Map from burn block index to the the reimbursed request.
     pub reimbursed_transactions: BTreeMap<u64, ReimbursedDeposit>,
 
+    /// Map from burn block index to the pending reimbursed withdrawal request.
+    ///
+    /// # Requirement
+    ///
+    /// A withdrawal request should only be reimbursed,
+    /// when it is certain that no Bitcoin transactions for that withdrawal will ever make it. That means,
+    /// 1. Either, the minter never issued a Bitcoin transaction including that withdrawal request;
+    /// 2. Or, it's guaranteed that such a transaction is no longer valid because some of its UTXOs
+    ///    have been used by another transaction that is considered finalized in the mean time.
```

Comment:
```suggestion
    /// A withdrawal request should only be reimbursed
    /// when it is certain that no Bitcoin transactions for that withdrawal will ever make it. That means,
    /// 1. Either the minter never issued a Bitcoin transaction including that withdrawal request;
    /// 2. Or it's guaranteed that such a transaction is no longer valid because some of its UTXOs
    ///    have been used by another transaction that is considered finalized in the meantime.
```